tests/integration_tests/test_defaults.py

```diff
@@ -0,0 +1,96 @@
+"""Tests here shouldn't require any sort of user data or instance setup."""
+
+import pytest
+
+from tests.integration_tests.instances import IntegrationInstance
+from tests.integration_tests.util import (
+    get_inactive_modules,
+    verify_clean_boot,
+    verify_clean_log,
+)
+
+
+@pytest.mark.ci
+class TestDefaults:
+    def test_clean_log(self, class_client: IntegrationInstance):
+        log = class_client.read_from_file("/var/log/cloud-init.log")
+        verify_clean_log(log, ignore_deprecations=False)
+        verify_clean_boot(class_client)
+
+    def test_inactive_modules(self, class_client: IntegrationInstance):
+        """Verify no errors, no deprecations and correct inactive modules in
+        log.
+        """
+        log = class_client.read_from_file("/var/log/cloud-init.log")
+
+        expected_inactive = {
+            "apt_pipelining",
+            "ansible",
+            "bootcmd",
+            "ca_certs",
+            "chef",
+            "disable_ec2_metadata",
+            "disk_setup",
+            "fan",
+            "grub_dpkg",
```

Comment:
Won't this fail with dailies on `oracular-` after #5842 is fixed?

Reply:
Updated